libsolidity/inlineasm/AsmScope.cpp

```diff
@@ -21,6 +21,7 @@
 #include <libsolidity/inlineasm/AsmScope.h>
 
 using namespace std;
+using namespace dev;
```

Comment:
If this is needed, then why is it only needed for windows?

Reply:
Hm - not so easy to find out what the standard actually says here - but it should indeed fix the build error and not cause any problems.

Reply:
``gcc`` doesn't allow accessing direct members of parent namespaces after importing a nested namespace, but it *does* allow to access sub-namespaces of parent namespaces, which is a bit weird... And apparently MSVC doesn't allow it... If anyone finds out what the standard actually says about this, I'd be interested to know :-).